### ISSUE TITLE
Feature: Localisation

### ISSUE THREAD

**Issue by Pdzly** [tgxn/lemmy-modder#116]
**What do you want to see?**
For  some people it is easier to use the page if it is in their native language.

**Additional context**
For React i18 Next is a good choice. 
https://react.i18next.com/

**Comment by tgxn** [comment]
this is currently low-priority for me, but we can definitely make a branch and start on pulling the strings out. (i'm assuming each string will need to be added to a centralized-json file or something where we can store each language)

on previous projects, we've used Weblate to organize translations. What have you used? 

https://www.ui-dev.in/2021/02/simplest-implementation-of-localization.html

**Comment by Pdzly** [comment]
Yeah i would not say it is high prio as features gets implemented, it doesnt make sense, but just having it in your backlog would be good. Yeah in per file languiage file e.g. en.json de.json

We just send it to a one person translator
But yeah i think weblate can be easy.
